src/python_testing/TC_LVL_2_3.py

```diff
@@ -81,7 +81,7 @@ def steps_TC_LVL_2_3(self) -> list[TestStep]:
                          "The third entry in reportedRemainingTimeValuesList is equal to 0")
                 ]
 
-    @per_endpoint_test(has_cluster(Clusters.LevelControl))
+    @run_for_each_matching_endpoint(has_cluster(Clusters.LevelControl))
```

Comment:
This should have docs about how `--int-arg forced_endpoint:N` is how you run on only one endpoint at a time for testing.

Reply:
Added to the assertion message.